base/interactiveutil.jl

```diff
@@ -257,7 +257,7 @@ and type signature to `io` which defaults to `STDOUT`. The ASTs are annotated in
 as to cause "non-leaf" types to be emphasized (if color is available, displayed in red).
 This serves as a warning of potential type instability. Not all non-leaf types are particularly
 problematic for performance, so the results need to be used judiciously.
-See [Manual](:ref:`man-code-warntype`) for more codeinformation.
+See [Manual](:ref:`man-code-warntype`) for more code information.
```

Comment:
Jeez, I'm messing things up, thought it was there in the original docstring. Sorry for being clumsy, will update again.